bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -86,18 +86,25 @@ spec:
           fi
 
           # And finally start the ovs-vswitchd now the DB is prepped
-          /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --no-ovsdb-server --system-id=random
+          /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --no-ovsdb-server --system-id=random --no-monitor
```

Comment:
@abhat so the only downside to using --no-monitor is if the process segfaults or something, do we have allt he necessary health-checks to kill the container when those processes die? we've got one for the vswitchd I think (openshift-sdn's healthcheck.go) but not for the dbserver. Or it could be the other way around.

Reply:
Liveness probe here will make sure that kubelet will kill the pod if ovsdb or ovs-vswitchd are not responsive. https://github.com/openshift/cluster-network-operator/blob/9fb694ed7ec901e382585e23b16f1e83ac8a3af6/bindata/network/openshift-sdn/sdn-ovs.yaml#L144. Will that be sufficient?

Reply:
Yeah I think that's fine.